packages/manager/modules/hub/src/components/user-panel/user-infos/template.html

```diff
@@ -0,0 +1,74 @@
+<div class="manager-hub-user-infos">
+    <span
+        class="manager-hub-user-infos_initials"
+        data-ng-class="{
+            'manager-hub-user-infos_initials-centered': !$ctrl.isExpanded,
+            'float-left': $ctrl.isExpanded
+          }"
+    >
+        <span data-ng-if="$ctrl.me" data-ng-bind="$ctrl.getNameInitials()">
```

Comment:
Maybe that nested span is not needed ? Can the ng-if and the ng-bind be applied on the parent ?